girder/models/item.py

```diff
@@ -396,7 +396,7 @@ def copyItem(self, srcItem, creator, name=None, folder=None,
         return newItem
 
     def fileList(self, doc, user=None, path='', includeMetadata=False,
-                 subpath=True, mimeFilter=None):
+                 subpath=True, mimeFilter=None, streamCallback=None):
```

Comment:
I think it would be better to break this up from a tuple into three separate parameters, then the types can be expressed more cleanly in the sphinx docs:

```
:param callback: A callback that will be called for each file found. If not set, defaults to the `download` method of the file model. This is always called with the file as the first argument.
:type callback: callable
:param callbackPargs: Optional additional positional arguments to be passed into the callback after the file argument.
:type callbackArgs: iterable
:param callbackKwargs: Optional additional keyword arguments to be passed into the callback.
:type callbackKwargs: dict
```